Gemfile

```diff
@@ -7,28 +7,28 @@ end
 
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 5.1.1'
+gem 'rails'
```

Comment:
One concern I have is that `app:update` needs to be run to keep everything in sync with the version of rails being used.  How possible or likely is it that future rails updates will cause the code to fail to build or run without subsequent app:update?  In that case, maybe this needs to remain locked in (but at ~> 6.1.X for now) and periodically updated for each X.Y release of Rails?

Reply:
The version is locked via `Gemfile.lock`. This lock would need to be removed / regenerated to cause any error. This should not be done for running the example app - let me know if there conflicting plans / guides / instructions.

Reply:
Right, as long as we leave Gemfile.lock in place, we should be fine.